hazelcast/src/test/java/com/hazelcast/map/EvictionTest.java

```diff
@@ -827,13 +828,7 @@ public void testIssue1085EvictionBackup() {
         IMap<Integer, Integer> map = instances[0].getMap(mapName);
 
         final CountDownLatch latch = new CountDownLatch(entryCount);
-        map.addEntryListener(new EntryAdapter<Integer, Integer>() {
-            @Override
-            public void entryEvicted(EntryEvent<Integer, Integer> event) {
-                super.entryEvicted(event);
-                latch.countDown();
-            }
-        }, false);
+        map.addEntryListener((EntryExpiredListener<Integer, Integer>) event -> latch.countDown(), false);
```

Comment:
Minor thing but maybe we should rename test names as well. testMapRecordEviction() -> testMapRecordExpiration() and testIssue1085ExpirationBackup().